ui_framework/src/components/form/select/select.js

```diff
@@ -0,0 +1,31 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+export const KuiSelect = ({ className, options, id, name, ...rest }) => {
+  const classes = classNames('kuiSelect', className);
+
+  return (
+    <select
+      id={id}
+      name={name}
+      className={classes}
+      {...rest}
+    >
+      {options.map((option, index) => {
+        return <option value={option.value} key={index}>{option.text}</option>;
+      })}
+    </select>
+  );
+};
+
+KuiSelect.propTypes = {
+  name: PropTypes.string.isRequired,
+  id: PropTypes.string,
+  options: PropTypes.arrayOf(React.PropTypes.object).isRequired,
+};
+
+
+
+
```

Comment:
Is there any way to set Vim to ensure there's only one empty line at the end of the file? https://www.google.com/search?q=vim+eof+newline&rlz=1C5CHFA_enUS695US695&oq=vim+eof+&aqs=chrome.3.69i57j0l5.5218j0j7&sourceid=chrome&ie=UTF-8